src/translations/en-US.json

```diff
@@ -194,6 +194,16 @@
         "comment": "",
         "limit": 0
     },
+    "visualization.ErrorMessageMissingMapboxToken": {
```

Comment:
could we reuse the text `visualization.ErrorMessageGeneric`?

Reply:
fixed